stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channel/add/group/select_name/AddGroupChannelSelectNameFragment.kt

```diff
@@ -0,0 +1,108 @@
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import io.getstream.chat.ui.sample.R
+import io.getstream.chat.ui.sample.common.initToolbar
+import io.getstream.chat.ui.sample.common.navigateSafely
+import io.getstream.chat.ui.sample.databinding.FragmentAddGroupChannelSelectNameBinding
+import io.getstream.chat.ui.sample.feature.channel.add.group.AddGroupChannelMembersSharedViewModel
+
+class AddGroupChannelSelectNameFragment : Fragment() {
+
+    private var _binding: FragmentAddGroupChannelSelectNameBinding? = null
+    private val binding get() = _binding!!
+    private val viewModelSelect: AddGroupChannelSelectNameViewModel by viewModels()
+    private val sharedMembersViewModel: AddGroupChannelMembersSharedViewModel by activityViewModels()
+    private val adapter = AddGroupChannelSelectNameMembersAdapter()
+    private var isCreateButtonEnabled: Boolean = false
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentAddGroupChannelSelectNameBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.menu_add_group_channel_select_name, menu)
+    }
+
+    override fun onPrepareOptionsMenu(menu: Menu) {
+        menu.findItem(R.id.createGroupChannelButton).actionView.findViewById<View>(R.id.addChannelButton).apply {
+            isEnabled = isCreateButtonEnabled
+            setOnClickListener {
+                sharedMembersViewModel.members.value?.let { members ->
+                    viewModelSelect.onEvent(
+                        AddGroupChannelSelectNameViewModel.Event.CreateChannel(
+                            name = binding.nameEditText.text.toString(),
+                            members = members
+                        )
+                    )
+                }
+            }
+        }
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initToolbar(binding.toolbar)
+        binding.membersRecyclerView.adapter = adapter
+        adapter.deleteMemberClickListener = AddGroupChannelSelectNameMembersAdapter.DeleteMemberClickListener {
+            sharedMembersViewModel.removeMember(it)
+        }
+        sharedMembersViewModel.members.observe(viewLifecycleOwner) { members ->
+            if (members.isEmpty()) {
+                findNavController().navigateUp()
+            } else {
+                binding.membersTitleTextView.text =
+                    getString(R.string.add_group_channel_select_name_members_title, members.size + 1)
```

Comment:
Are we adding `+1` here to count the current user? Feels odd to me on the UI, took me a while to guess why the number is higher than the number of users I selected:

![image](https://user-images.githubusercontent.com/12054216/100740120-78594180-33d8-11eb-88f1-aed949019f68.png)

I don't see this kind of mismatch on the designs.

Reply:
True, I assumed that we have to add +1 because of currentUser but it doesn't make sense